peekingduck/cli.py

```diff
@@ -16,78 +16,23 @@
 CLI functions for PeekingDuck.
```

Comment:
This is a much better way to create custom nodes and we should get users to use this method as the default. Let's update the custom nodes documentation accordingly, but still provide instructions for the old way in case users still need to do it the old way e.g. manually copy some files over to the right directories. 

We can also take this opportunity to revamp any sections here that were previously unclear (if any).

Reply:
Added instructions https://github.com/aimakerspace/PeekingDuck/pull/525/commits/a74e4f62faabf27b6b30c179406747bddba60bec

Recommends using `create-node` by making it the default Step 3 and manual creation the Alternative Step 3.

Also included `<project_name>` in the various directory structures for clarity.

Reply:
1) The example config file and python script for `csv_writer` is now under "Alternative Step 3", but they should be copied regardless of the method used. Can these be moved to a "common section"?
2) There are a couple of "alternative" steps which can be confusing and make the docs more lengthy - considering point (1), and rethinking about this, perhaps we should remove the "alternative step 3" section altogether? Instead, we can write at the end of the directory structure that users can also manually create the folders and files as per the directory structure shows if they choose not to use the CLI method

Let me know what you think? After making these changes, the whole markdown needs to sound coherent as a whole.

Reply:
Ok, I think we can go with point (2)

Reply:
Looks good now!